.gitlab-ci.yml

```diff
@@ -20,6 +20,17 @@ include:
     ref: latest
     file: 'front-end-publish-npm.yml'
 
+type-check:
+  retry: 2
```

Comment:
I don't think we need retries do we?  This isn't a huge deal, but type checking shouldn't be flakey. Did you come across an actual need for this?

Reply:
I can remove it: I just copied this from another step I found.